### DISCUSSION TITLE
FPS Drop just from having OBS open, Please HELP

### DISCUSSION THREAD

**Discussion by Boothy26** [obsproject/obs-studio#8222] [Development Questions]
Hi,

Been troubleshooting this for weeks, done the basics turn off windows game mode run as admin etc etc...
Running an RTX3080 with i9 11900k if that helps.
Samsung G9 240hz main monitor, top monitor capped at 60hz.

As per the video attached with an unlisted stream, is there a way to turn down the quality of obs window or is it best to just keep it minimized. Disabling window preview doesnt really change anything, but as you can see just minimising the program has a major fps hit.

https://user-images.githubusercontent.com/123875410/216994340-6d9f02fc-8059-4f35-bd97-2f58ffb67cfa.mp4

**Comment by notr1ch** [comment]
This forum is for development. Please use the forums or Discord for support issues.
